bridges/snowbridge/pallets/inbound-queue-v2/src/lib.rs

```diff
@@ -215,9 +166,9 @@ pub mod pallet {
 		}
 	}
 
-	/// The current nonce for each channel
+	/// The nonce of the message been processed or not
 	#[pallet::storage]
-	pub type Nonce<T: Config> = StorageMap<_, Twox64Concat, ChannelId, u64, ValueQuery>;
+	pub type Nonce<T: Config> = StorageMap<_, Identity, u64, bool, ValueQuery>;
```

Comment:
The idiomatic way to implement a _Set_ data structure using storage maps is to use the empty value `()` as the storage value:
```suggestion
	pub type Nonce<T: Config> = StorageMap<_, Identity, u64, (), ValueQuery>;
```

Reply:
Now irrelevant with the sparse bitmap data structure: https://github.com/claravanstaden/polkadot-sdk/pull/3#discussion_r1851556972